### PR TITLE
chore(babel): Bump @babel dependencies

### PR DESCRIPTION
**Pull request by priscilawebdev**
Bump @babel dependencies

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -181,6 +186,28 @@
     semver "^5.4.1"
     source-map "^0.5.0"
 
+"@babel/core@~7.12.3":
```
Comment (markstory) [inline yarn.lock:189]: Are we able to consolidate versions so that we don't install multiple versions of babel libraries?

**Comment by billyvg** [comment]
We also need to bump these in getsentry

File: yarn.lock (hunk 1 above)
Comment (priscilawebdev) [reply yarn.lock:189]: Please what do you mean? I'm not sure if I understood

**Comment by priscilawebdev** [comment]
@billy I've created a PR in getsentry https://github.com/getsentry/getsentry/pull/4611/files

**Review by markstory — APPROVED** [review]

**Review by billyvg — APPROVED** [review]
Can we merge this separately from `react` upgrade (maybe merge it a few days after we merge `react`)

**Pull request merged**